pkgs/by-name/go/go-mtree/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "gomtree";
```

Comment:
Fedora packages this as gomtree so I suggest we drop the dash in the attribute name (directory name in this case) as well: https://repology.org/project/gomtree/versions

Reply:
changed it to `gomtree`